src/common/viz/FigureExtractor.js

```diff
@@ -86,8 +86,34 @@ define(['./Utils'], function (Utils) {
         return desc;
     };
 
-    FigureExtractor.prototype[EXTRACTORS.PLOT3D] = function(/*node*/) {
-        throw new Error('Not Implemented yet');
+    FigureExtractor.prototype[EXTRACTORS.PLOT3D] = function(node) {
+        const id = node.getId(),
+            graphId = node.getParentId(),
+            execId = this.getExecutionId(node);
+        let desc;
+
+        desc = {
+            id: id,
+            execId: execId,
```

Comment:
It might be nice to reduce the code duplication between 2D and 3D plots. The only gotcha is that 3D plots don't support images. Maybe they could be refactored to add a method for the shared abstract base class in the metamodel and both could call it...